conformance/lang/expressions/unary-expression/unary-minus-expression.balt

```diff
@@ -0,0 +1,984 @@
+Test-Case: output
+Description: Test unary negative expression for positive integers, no overflow scenarios.
+Label: DecimalNumber, int, unary-minus
+
+function init() {
+    int a = 1;
+    int b = 101;
+    int c = 0;
+    int d = 9223372036854775807;
+    io:println(-a); // @output -1
+    io:println(-b); // @output -101
+    io:println(-c); // @output 0
+    io:println(-d); // @output -9223372036854775807
+}
```

Comment:
Lest add some tests with HexIntLiteral eg: `-0xAB55`

Reply:
Updated in 4e28ce2